ci-operator/step-registry/sdn/conf/multi-manifest/sdn-conf-multi-manifest-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
+  as: sdn-conf-multi-manifest
+  from: base
+  commands: sdn-conf-multi-manifest-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  documentation: |-
+    The sdn multi manifest step adds the configuration for sdn multi tenant networking to set of manifests for installation
```

Comment:
nits: "sdn" -> "SND", "multi manifest" -> "multi-manifest", "multi tenant" -> "multi-tenant", and trailing period.  Also probably mention the `manifest_cluster-network-03-config.yml` file and the fact that ipi-install-install is the intended consumer.

Reply:
OK, did the single-manifest file as well.

Reply:
Its nice to clean this up. Thanks for the comments.